index.bs

```diff
@@ -1228,7 +1234,7 @@ Consider the following:
 
 This is equivalent to the following postfix notation (parentheses for clarity):
 
-<math><msub><mi>sample</mi><mi>output</mi></msub><mo>=</mo><mo>(</mo><mn>256</mn><mspace width="1ch"/><msub><mi>sample</mi><mn>1</mn></msub><mo>×</mo><mo>)</mo><msub><mi>sample</mi><mn>2</mn></msub><mo>+</mo></math>
+<math><msub><mi>sample</mi><mi>output</mi></msub><mo>=</mo><mo>(</mo><mn>256</mn><mspace width="0.5em"/><msub><mi>sample</mi><mn>1</mn></msub><mo>×</mo><mo>)</mo><mspace width="0.5em"/><msub><mi>sample</mi><mn>2</mn></msub><mo>+</mo></math>
```

Comment:
`"1ch"` does not seem to work in Safari. The space simply disappears. `"0.5em"` seems to work in both Chrome and Safari and gives roughly the same space.

Reply:
Yeah, it's weird. Maybe it doesn't work correctly in <math> tags but works in other CSS stuff?

Reply:
Good question. That looks like it should be supported in Safari though. I pinged the WebKit folks to see if they know about it.

Reply:
Turns out it's an issue in Firefox as well. WebKit folks opened an issue for it:
https://bugs.webkit.org/show_bug.cgi?id=281232